app/assets/javascripts/mountain_view.js.erb

```diff
@@ -1,5 +1,7 @@
-<% depend_on Rails.root.join('app', 'components').to_s %>
-<% Dir.glob(Rails.root.join('app', 'components', '*')) do |component_dir|
-  component = File.basename component_dir
-  require_asset "#{component}/#{component}"
-end %>
+<% if Dir.exists?(Rails.root.join("app", "components") ) %>
+  <% depend_on Rails.root.join('app', 'components').to_s %>
```

Comment:
@kitop I get this error if I put this line above the if:

Internal Server Error

couldn't find file '/Users/nachogutierrez/Projects/mountain_view/test/dummy/app/components' (in /Users/nachogutierrez/Projects/mountain_view/app/assets/stylesheets/mountain_view.css.erb)